src/citrine/resources/predictor.py

```diff
@@ -147,6 +153,22 @@ def convert_to_graph(self,
                 raise exc
         return self.build(entity)
 
+    def is_stale(self,
+                 uid: Union[UUID, str],
+                 *,
+                 version: Union[int, str] = MOST_RECENT_VER) -> bool:
+        path = self._construct_path(uid, version, "is-stale")
+        response = self.session.get_resource(path, version=self._api_version)
+        return response["is_stale"]
+
+    def retrain_stale(self,
+                      uid: Union[UUID, str],
+                      *,
+                      version: Union[int, str] = MOST_RECENT_VER) -> Predictor:
+        path = self._construct_path(uid, version, "retrain-stale")
+        entity = self.session.put_resource(path, {}, version=self._api_version)
```

Comment:
It's weird that we have to manually put in the API version.  Seems like we should have cleaned this up to defaultly use the `self._api_version`.